### PR TITLE
Feature/error pages

### PR DESCRIPTION
**Pull request by Moro-Code**
# Description 📝

- created error page component
- added error page component to home page
- added 404 page

Screen Shot 2020-12-08 at 2 29 25 PM
Screen Shot 2020-12-08 at 2 29 44 PM

## Checklist ✅
- [x] created story for all visual components
- [x] created or modified necessary unit tests
- [ ] created or modified e2e tests
- [ ] modified README or any applicable documentation on the changes you made

### CODE REVIEW

File: src/components/organisms/ErrorPage.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ * page component for errors
+ */
+export function ErrorPage(props) {
+  return (
+    <Page>
+      <main className="w-full h-full flex flex-col justify-center items-center font-sans">
+        <h2 className="text-3xl md:text-6xl">{props.errorTitle}</h2>
+        <pre className="w-full h-auto bg-gray-500 rounded-lg border border-red-400 whitespace-pre-wrap">
+          {typeof props.error === "object"
+            ? JSON.stringify(props.error, null, 2)
```
Comment (s-laugh) [inline src/components/organisms/ErrorPage.js:18]: Is there a test for the error as an object?

File: src/pages/home.js [hunk 2]
```diff
@@ -83,6 +85,18 @@ export function Home() {
       : dispatch(deselectBenefitActionCreator(benefitId));
   };
 
+  if (fetchBenefitsFailed || fetchBenefitsCountFailed) {
```
Comment (s-laugh) [inline src/pages/home.js:88]: Not-blocking: Is this going to get lengthily if more error types are captured? If so, should a more generic handler be looked at?

**Review by s-laugh — COMMENTED** [review]
Looks really good, just noticed a possible missing test, and wondering about the maintainability of the error handling.

File: src/pages/home.js (hunk 2 above)
Comment (Moro-Code) [reply src/pages/home.js:88]: We can look at that down the road if need be. Think this works for now

File: src/components/organisms/ErrorPage.js (hunk 1 above)
Comment (Moro-Code) [reply src/components/organisms/ErrorPage.js:18]: Should add that

File: src/pages/home.js (hunk 2 above)
Comment (s-laugh) [reply src/pages/home.js:88]: Agreed

**Pull request merged**